ledger/ledger-api-tests/infrastructure/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Synchronize.scala

```diff
@@ -41,7 +41,7 @@ object Synchronize {
       party: Party,
       contractId: Primitive.ContractId[T],
   )(implicit ec: ExecutionContext): Future[Unit] = {
-    Eventually.eventually {
+    Eventually.eventually("Wait for contract") {
```

Comment:
would be good to import `Eventually` here so it reads better ;) 

Wait for contract => Wait for contract to become active